Wordle.Api/Wordle.Api/Migrations/20230528165002_Identity.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
```

Comment:
for migrations, if you've not changed your local db, you may need to wipe it and it's migrations to conform with these new classes